pkgs/top-level/perl-packages.nix

```diff
@@ -14646,6 +14646,21 @@ let
     };
   };
 
+  TermShell = buildPerlModule rec {
+    name = "Term-Shell-0.10";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/S/SH/SHLOMIF/${name}.tar.gz";
+      sha256 = "7d1f824c2db22769b60000b5b9ca2ad469c154939f9ec1cd3f0e06e9c967dda3";
+    };
+    buildInputs = [ ModuleBuild ];
+    propagatedBuildInputs = [ TermReadKey TextAutoformat ];
+    meta = {
```

Comment:
with stdenv.lib

Reply:
perhaps ```ModuleBuild``` is not needed as ```buildPerlModule``` adds it.
LGTM, anyway, ```cpan2nix``` will fix ```perl-packages.nix``` issues on the next run

Reply:
I added with stdenv.lib, and removed the ModuleBuild (even if I think cpan2nix added it for me, but not sure anymore). Thanks!